.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Global rule:
+*           @sethiay @vadlakondaswetha
```

Comment:
does this work on all branches or only master?

Reply:
We need to add a branch protection rule to configure it for other branches. Currently, only rule is for master branch.

Reply:
can you add it for sept_release branch also

Reply:
Is it possible to specify branches using regex (e.g. master|*_release) ?

Reply:
Yeah, we can specify the regex. There is no right way to combine both branches to a single rule.

Hence, I have created two rules:
(a) master
(b) *_release - (we need to make sure to create a branch ending with _release).

Reply:
i am sorry, i can't where you added rule for sept_release branch also?

Reply:
https://screenshot.googleplex.com/9RRt9CmEPmXny5T